drivers/unix/file_access_unix.cpp

```diff
@@ -44,7 +44,7 @@
 #include <unistd.h>
 #endif
 
-#ifdef MSVC
+#ifdef _MSC_VER
```

Comment:
I wonder if that isn't dead code btw. In what condition would we build this with MSVC since it's under `UNIX_ENABLED`?

Reply:
I removed all that code, which seemed partially copied from FileAccessWindows.